package.json

```diff
@@ -31,15 +31,15 @@
   },
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
-    "@edx/frontend-component-footer": "12.2.0",
-    "@edx/frontend-component-header": "4.6.0",
-    "@edx/frontend-platform": "5.5.4",
-    "@edx/paragon": "^20.44.0",
+    "@edx/frontend-component-footer": "13.0.2",
+    "@edx/frontend-component-header": "5.0.2",
+    "@edx/frontend-platform": "7.1.0",
```

Comment:
Are these imports supposed to stay `@edx`?

Reply:
Yes. As of writing, all 3 are still being published to `@edx`

https://github.com/openedx/frontend-component-footer/blob/62bf038a06feac7865e558205b29bc726e9594c4/package.json#L2
https://github.com/openedx/frontend-component-header/blob/3b2a2bfa9576926fcf6278b222835d68752bba0e/package.json#L2
https://github.com/openedx/frontend-platform/blob/f0e120244bb367b524c72ba30ef7e09c6b2cbcfc/package.json#L2

Reply:
Thanks @brian-smith-tcril. Any chance we could merge this pull request?

cc: @zubair-ce07

Reply:
On it.